src/dune_init.ml

```diff
@@ -126,8 +124,8 @@ module File = struct
         | Some (a, b) ->
           die "Updating existing stanzas is not yet supported.\n\
                A preexisting dune stanza conflicts with a generated stanza:\n\n\
-               Generated stanza:\n%s\n\nPre-existing stanza:\n%s"
-            (to_string a) (to_string b)
+               Generated stanza:\n%a\n\nPre-existing stanza:\nas"
```

Comment:
Shouldn't the last conversion read `%a`?

Reply:
Yes, that's a good point. It's quite annoying that `die` swallows things this way because its return type is polymorphic